rules.d/50-udev-default.rules.in

```diff
@@ -17,6 +17,15 @@ SUBSYSTEM=="rtc", KERNEL=="rtc0", SYMLINK+="rtc", OPTIONS+="link_priority=-100"
 SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", IMPORT{builtin}="usb_id", IMPORT{builtin}="hwdb --subsystem=usb"
 ENV{MODALIAS}!="", IMPORT{builtin}="hwdb --subsystem=$env{SUBSYSTEM}"
 
+# Before c43ff248f94266cfc93e300a2d3d163ed805e55b, the following line in
+# 60-drm.rules also sets ID_PATH for almost all pci, usb, and platform devices:
+####
+# ACTION!="remove", SUBSYSTEM=="drm", SUBSYSTEMS=="pci|usb|platform", IMPORT{builtin}="path_id"
+####
+# Unfortunately, some existing rules rely on the unexpected behavior.
+# To keep the backward compatibility, let's set ID_PATH for them.
+SUBSYSTEM=="pci|usb|platform", SUBSYSTEMS=="pci|usb|platform", IMPORT{builtin}="path_id"
```

Comment:
Does `SUBSYSTEMS` also match the device itself (or just parents)? Because if it does, then we wouldn't need both `SUBSYSTEM` and `SUBSYSTEMS`.

Reply:
Indeed. `SUBSYSTEMS` also matches the device itself. I forgot that. Thanks.